modules/aws/s3.go

```diff
@@ -230,6 +287,30 @@ func AssertS3BucketExistsE(t *testing.T, region string, name string) error {
 	return err
 }
 
+// AssertS3BucketVersioningExists checks if the given S3 bucket has a versioning configuration enabled and returns an error if it does not.
+func AssertS3BucketVersioningExists(t *testing.T, region string, bucketName string) {
+	err := AssertS3BucketVersioningExistsE(t, region, bucketName)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+// AssertS3BucketVersioningExistsE checks if the given S3 bucket has a versioning configuration enabled and returns an error if it does not.
+func AssertS3BucketVersioningExistsE(t *testing.T, region string, bucketName string) error {
+	status, err := GetS3BucketVersioningE(t, region, bucketName)
+	if err != nil {
+		return err
+	}
+
+	errorMsg := errors.New("bucket versioning is not enabled")
```

Comment:
Can you create an [error type](https://github.com/gruntwork-io/terratest/blob/master/modules/aws/errors.go)? It makes checking whether the right error is being returned easier. E.g to differentiate between this error and an error from trying to retrieve the bucket.

Reply:
Implemented